subsys/bluetooth/shell/bt.c

```diff
@@ -3714,6 +3761,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(bt_cmds,
 		      cmd_auth, 2, 0),
 	SHELL_CMD_ARG(auth-cancel, NULL, HELP_NONE, cmd_auth_cancel, 1, 0),
 	SHELL_CMD_ARG(auth-passkey, NULL, "<passkey>", cmd_auth_passkey, 2, 0),
+#if defined(CONFIG_BT_PASSKEY_KEYPRESS)
+	SHELL_CMD_ARG(auth-passkey-notify, NULL, "<type>",
+		      cmd_auth_passkey_notify, 2, 0),
```

Comment:
I wonder if it would actually be possible to do the passkey notifications when entering the passkey in `cmd_auth_passkey` :o

Reply:
It's a good idea. I talked with @jori-nordic, and we decided we can do it in a later PR as an option to `cmd_auth_passkey`. Can I resolve this conversation?